codeigniter4/framework/CVE-2022-21647.yaml

```diff
@@ -0,0 +1,8 @@
+title:     "CVE-2022-21647: Deserialization of Untrusted Data in Codeigniter4"
+link:      https://github.com/codeigniter4/CodeIgniter4/security/advisories/GHSA-w6jr-wj64-mc9x
+cve:       CVE-2022-21647
+branches:
+    master:
+        time:     2022-01-04 00:59:31
+        versions: ['<4.1.6']
+reference: composer://codeigniter4/framework
```

Comment:
What about the `codeigniter4/codeigniter4` package?

Reply:
Thank you. We don't have `codeigniter4/codeigniter4`.
https://packagist.org/packages/codeigniter4/